scalding-core/src/main/scala/com/twitter/package.scala

```diff
@@ -34,7 +34,7 @@ package object scalding {
   /**
    * Make sure this is in sync with version.sbt
    */
-  val scaldingVersion: String = "0.17.0"
+  val scaldingVersion: String = "0.17.2"
```

Comment:
we should automate this: we could have the build create a file and we could import that file to get the version (I have seen people do something similar for the git-sha which would also be nice.